GitSearch/Application/RepoInfo/RepoInfoCell.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  RepoInfoCellView.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/3/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import UIKit
+
+class RepoInfoCell: UITableViewCell {
+    
+    let title: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: 13)
+        label.textColor = #colorLiteral(red: 0.3921568627, green: 0.3921568627, blue: 0.3921568627, alpha: 1)
+        return label
+    }()
+    
+    let propertyValue: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.boldSystemFont(ofSize: 20)
+        label.numberOfLines = 0
+        return label
+        }()
+    
+    let separator: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = #colorLiteral(red: 0.8823529412, green: 0.8823529412, blue: 0.8823529412, alpha: 1)
+        return view
+        }()
+    
+    func setupSubviews() {
+        contentView.addSubview(title)
+        contentView.addSubview(propertyValue)
+        contentView.addSubview(separator)
+        
+        let borderSize: CGFloat = 25
+        let borderSizeShift: CGFloat = 5
+        let separatorHeight: CGFloat = 1
+
+        let constraints = [
+            title.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16),
+            title.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: borderSize + borderSizeShift),
+            title.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -borderSize*2),
```

Comment:
It seems that this logic is incorrect. Do you really want this label have trailing margin twice as big as leading? Otherwise, `borderSize` should be the constant to use.

Reply:
That was an idea to set right border twice bigger than left - call it design feature.
But you right, borderSize should be in constant class because of using in RepoInfoCell.swift and RepoInfoDescriptionCell.swift
I will do it in next PR, when constans class will be in master.